modules/release-notes/partials/docs-server-7.2.7-release-note.adoc

```diff
@@ -1,11 +1,22 @@
 [#release-727]
 == Release 7.2.7 (March 2025)
 
-Couchbase Server 7.2.7 was released in March 2025.
-This maintenance release contains the following fixes.
+[#deprecated-features-and-platforms-720]
+=== Deprecated and Removed Features and Platforms
+
+* The following operating systems are no longer supported:
```

Comment:
We're only deprecating in 7.2/7.6 for now (though we may remove them in a later release due to build/packaging issues if we need to), so these are still supported at this time.